requirements/requirements_test.txt

```diff
@@ -32,32 +42,25 @@ flake8-mock==0.3
 flake8-mutable==1.1.0
 # Checks for TODO statements (and similar)
 flake8-todo==0.7
-pycodestyle==2.3.1
 # Checks for uses of double quotes (without containing single quotes)
 flake8-quotes==0.9.0
 # Checks for uses of old-style string formatting (using % instead of .format())
 flake8-pep3101==1.0
 # Checks for the overriding of builtins
-git+ssh://git@github.com/vikingco/flake8-builtins.git@1.1#egg=flake8-builtins
+flake8-builtins-unleashed==1.3.1
 # Checks for uses of Django-style asserts
--e git+ssh://git@github.com/vikingco/flake8-pytest.git@1.1#egg=flake8-pytest
+flake8-pytest==1.3
 # Checks for bad imports of fake_time (should come from libfaketime_tz_wrapper instead of libfaketime)
--e git+ssh://git@github.com/vikingco/flake8-libfaketime.git@1.0#egg=flake8-libfaketime
+flake8-libfaketime==1.1
 # Checks for module-level imports
-git+ssh://git@github.com/vikingco/flake8-module-import.git@1.0#egg=flake8-module-imports
+flake8-module-imports==1.1
 # Checks for usages of django.translation.activate()
-git+ssh://git@github.com/vikingco/flake8-translation-activate.git@1.0.1#egg=flake8-translation-activate
+flake8-translation-activate==1.0.2
 # Checks for imports with an alias of _
-git+ssh://git@github.com/vikingco/flake8-ugettext-alias.git@1.0#egg=flake8-ugettext-alias
+flake8-ugettext-alias==1.1
 # Checks for imports of the User model (instead of get_user_model() or settings.AUTH_USER_MODEL)
-git+ssh://git@github.com/vikingco/flake8-user-model.git@1.0#egg=flake8-user-model
-
-# Coverage uploads
-coveralls
-coverage==3.7.1
-docopt==0.6.2
-requests==2.9.1
+flake8-user-model==1.1
 
 # StatsD allows us to push monitoring data to a Graphite server
-git+ssh://git@github.com/vikingco/django-statsd.git@0.4.0#egg=django-statsd-mozilla
```

Comment:
Don't we want statsd in the actual requirements instead of test_requirements?

Reply:
Looks like an optional requirement to me: https://github.com/vikingco/django-triggers/search?utf8=%E2%9C%93&q=statsd&type=

Reply:
:+1: